tvtk/tests/test_tvtk.py

```diff
@@ -337,19 +330,14 @@ def test_property(self):
         self.assertEqual(p.color, (0.5, 0.5, 0.5))
 
     def test_points_lookup(self):
-        """ Test if points can be looked up with both int and long keys.
-            Fixes GH Issue 173.
+        """ Test if points can be looked up.
```

Comment:
```suggestion
        """ Test if points can be looked up. Fixes GH Issue 173.
```

Reply:
I actually removed the mention of 173 on purpose as the test is now not really related to the issue:
https://github.com/enthought/mayavi/issues/173

ie the test has nothing to do with `long` anymore.

I am happy to leave the comment though if you think that makes more sense

Reply:
ahh. thanks for the context. resolving.